Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionProcess.cs

```diff
@@ -15,6 +15,23 @@ private IdentityDeletionProcess()
     {
     }
 
+    public static IdentityDeletionProcess CreateDeletionProcessAsSupport(IdentityAddress createdBy)
+    {
+        return new IdentityDeletionProcess(createdBy, DeletionProcessStatus.WaitingForApproval);
+    }
```

Comment:
Now there are 2 ways to create a deletion process. One with the constructor and one with a factory method. That's bad. I'm okay with using a factory method, but then you should refactor existing code accordingly.